programs/Makefile

```diff
@@ -67,6 +69,7 @@ APPS =	aes/aescrypt2$(EXEXT)		aes/crypt_and_hash$(EXEXT)	\
 	random/gen_random_ctr_drbg$(EXEXT)				\
 	test/ssl_cert_test$(EXEXT)	test/benchmark$(EXEXT)		\
 	test/selftest$(EXEXT)		test/udp_proxy$(EXEXT)		\
+	test/header_test$(EXEXT)								\
```

Comment:
I don't think we should build this program by default. People who only have a C compiler and no C++ compiler, or who've set `CC` but not `CXX`, should be able to run `make`.

This applies to other build systems as well.

Reply:
Fixed.